app/controllers/concerns/reports/review_score_details_report.rb

```diff
@@ -317,8 +318,15 @@ def review_score_details_csv_rows
           conclusion_review.conclusion,
           conclusion_review.evolution,
           conclusion_review.review.business_unit.to_s,
-          conclusion_review.review.manual_score
+          conclusion_review.review.manual_score,
+          user_assignment_auditor(conclusion_review.review)
         ]
       end
     end
+
+    def user_assignment_auditor review
+      auditors = review.review_user_assignments.select &:auditor?
+
+      auditors.map(&:user).pluck(:user).join('; ')
```

Comment:
```suggestion
      auditors.map(&:user).join '; '
```
El `to_s` es el atributo `user`.